.build/traefik/http.yml

```diff
@@ -1,9 +1,14 @@
 http:
   services:
-    ipfs-http-adapter:
```

Comment:
Is the ipfs adapter still exposed?

Reply:
It is and it must be exposed in order to make IPFS work with Alkemio Server outside of the docker-compose network.

Reply:
This PR makes it look like the ipfs adapter is being removed